lib_static/open_project/configuration/helpers.rb

```diff
@@ -103,7 +103,8 @@ def remote_storage_hosts
         [
           fog_credentials[:host],
           remote_storage_upload_host,
-          remote_storage_download_host
+          remote_storage_download_host,
+          "nextcloud.local"
```

Comment:
TODO: Remove that

Reply:
This change makes these spec fail:

```
rspec ./spec/requests/auth/oauth_login_csp_spec.rb:54 # CSP appends on login form from oauth with redirect-uri being a custom scheme appends given CSP appends from flash
rspec ./spec/requests/auth/oauth_login_csp_spec.rb:40 # CSP appends on login form from oauth when login required appends given CSP appends from flash
```

Reply:
Removed.

Functionality now disabled. Will be active once https://community.openproject.org/projects/openproject/work_packages/45474/activity is done.